windows-agent/internal/daemon/testutils/testutils.go

```diff
@@ -0,0 +1,79 @@
+// Package testutils exports test helpers to be used in other packages that need to change internal behaviors of the daemon.
+package testutils
```

Comment:
if you create the subpackage named `daemontests` that can move there too :)

Actually, maybe testutils is a better naming for it and we should follow that convention (that forces though to use an alias when importing it). Which one do ou prefer?

Reply:
I think testutils is less invasive, easy to filter out of coverage reports (already done actually) and can have more test-related things that we can export or not to higher levels.

Reply:
Another option to ensure we have the same alias everywhere when importing it: maybe the subdirectory can be named `testutils` but the package name will still be something related to the parent package, like `daemontestutils`?
That way, when import a name that will be autoalias to the new name. Having long name there, for tests, don’t seem to undermine the gain by having this explicitely. Tell me what you think.

Reply:
That's better than the alias. We could also name the directory as `daemontestutils`  and, as we `grep -hv -e "testutils"` when filtering the coverage reports it will still match. Long names for packages are not the most idiomatic way in Go, but as you said, only for tests won't harm. I'm all for it.